modular_citadel/code/modules/reagents/reagents/cit_reagents.dm

```diff
@@ -37,11 +37,6 @@
 	..()
 	dir = pick(1,2,4,8)
 
-/datum/reagent/consumable/semen/reaction_turf(turf/T, reac_volume)
```

Comment:
This completely removes the ability to splash semen on turfs. Fix the runtime properly.

Reply:
See line 13 for the intended version of the proc. This seems like an older one which hasn't been properly removed before to me.

Reply:
Works fine for me, chief.
@deathride58 https://imgur.com/lfnSiKv8